tests/test_grid_fees.py

```
@@ -69,3 +69,103 @@ def test_breakdown_uses_snap_rate():
+# --- Vollständige Netzentgelt-Tabelle --------------------------------------- #
+#
+# Bisher wurde ausschließlich Wien ausgewertet; die übrigen 13 Netzgebiete waren
+# unbelegt. Es sind von Hand abgetippte, jährlich zu erneuernde Tarifwerte – der
+# realistische Fehler ist der Zahlendreher, und er verschiebt still den
+# Gesamtpreis jedes Nutzers in genau dieser Zone.
+#
+# Zwei Prüfungen mit unterschiedlicher Aussagekraft:
+#
+# 1. ERWARTETE_ZONEN ist eine bewusste Zweitschrift der Tabelle aus
+#    `grid_fees.py` (Netzentgelte 2026, Netzebene 7, Viertelstundenmessung, ohne
+#    Leistungsmessung; netto in ct/kWh). Sie ist damit KEINE unabhängige Quelle –
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)